udp-ballerina/listener.bal

```diff
@@ -88,9 +88,9 @@ public class Listener {
 #                runs without connecting to a remote host.
 # + localHost - The interface for the server to be bound.
 public type ListenerConfig record {
-    string? remoteHost = ();
-    int? remotePort = ();
-    string? localHost = ();
+    string remoteHost?;
```

Comment:
Don't you have to update the java code accordingly ?

Reply:
Old implementation compatible with this change